app/jobs/publish_registration_metrics.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class PublishRegistrationMetrics < WcaCronjob
+  before_enqueue do
+    # This information is not useful for us in staging
+    throw :abort unless EnvConfig.WCA_LIVE_SITE?
+  end
+
+  def record_last_60_minutes_registrations
+    count = Registration.where('created_at >= ?', 60.minutes.ago).count
```

Comment:
Fun fact: You can even map this to a AR native query! `Registration.where(created_at: 60.minutes.ago..)` works because Ruby supports open-ended ranges as a language feature: `a..` implicitly evaluates to `a..INFINITY` and Rails is smart enough to detect the infinity and map it to `>=` in SQL

Reply:
Sidenote: We do not use this feature consistently throughout the codebase because it is _relatively_ new (since Rails 6 or so) and we have queries which definitely predate that.

Reply:
I think rubocop-rails would be able to figure that out

Reply:
Yes, but last time we tried to introduce it it suggested like a gazillion changes which would have easily blown the 1000 line diff mark